composer.json

```diff
@@ -52,11 +52,21 @@
 		"dealerdirect/phpcodesniffer-composer-installer": "^0.4.3 || This Composer plugin will sort out the PHPCS 'installed_paths' automatically."
 	},
 	"scripts"    : {
-		"install-standards"    : "@php ./vendor/squizlabs/php_codesniffer/bin/phpcs --config-set installed_paths ../../..,../../wp-coding-standards/wpcs,../../phpcompatibility/phpcompatibility-wp,../../phpcompatibility/phpcompatibility-paragonie",
-		"run-tests"            : "@php ./vendor/phpunit/phpunit/phpunit --filter WPThemeReview ./vendor/squizlabs/php_codesniffer/tests/AllTests.php",
-		"phpcs-i"              : "@php ./vendor/squizlabs/php_codesniffer/bin/phpcs -i",
-		"check-cs"             : "@php ./vendor/squizlabs/php_codesniffer/bin/phpcs",
-		"fix-cs"               : "@php ./vendor/squizlabs/php_codesniffer/bin/phpcbf"
+		"install-standards": [
+			"@php ./vendor/squizlabs/php_codesniffer/bin/phpcs --config-set installed_paths ../../..,../../wp-coding-standards/wpcs,../../phpcompatibility/phpcompatibility-wp,../../phpcompatibility/phpcompatibility-paragonie"
+		],
+		"run-tests": [
```

Comment:
If we revert this to just a string (since we are not executing multiple commands here), will this fix the Travis failure?

Reply:
No, it won't.

I've just had a look and most of the Travis failure was just Travis being weird, the rest of it was me forgetting to update the minimum supported PHPCS version in the Travis script. Oops :flushed: 

I've fixed up the initial Composer update commit now to include the change to the Travis script and I expect the build will now pass.